src/Keboola/StorageApi/Client.php

```diff
@@ -1370,15 +1370,16 @@ public function getTable($tableId)
      * Drop a table
      *
      * @param string $tableId
-     * @param array $options - (bool) force
+     * @param array $options - (bool) force, (bool) async (default true)
      * @return mixed|string
      */
-    public function dropTable($tableId, $options = [])
+    public function dropTable($tableId, $options = ['async' => true])
```

Comment:
nechci ten async natvrdo vynucovat (jen dat default), protoze asi ho porad chceme mit pokryty testy, takze jsem to zatim vyresil takto.

Reply:
bude to v minor verzi a je to na urovni klienta. ta operace v klientovi (z pohledu dropTable metody ) se bude chovat stejne

Reply:
:+1: